docs/source/tutorials/instrument-modeling/contingent-claims-instrument.rst

```diff
@@ -26,48 +27,63 @@ rate instrument:
   :start-after: -- FIXED_RATE_BOND_TEMPLATE_BEGIN
   :end-before: -- FIXED_RATE_BOND_TEMPLATE_END
 
-The ``Contingent Claims`` tree is not part the template. Instead, it will be created dynamically, as
-described in the next sections.
+These template variables describe the payoff of the fixed rate bond.
+They will be used to create a :doc:`Contingent Claims <../../concepts/contingent-claims>` tree,
+which is the internal representation used for modelling and lifecycling in Daml Finance.
+Note that this tree is not part the template above.
+Instead, it will be created dynamically, as described in the next sections.
```

Comment:
maybe you don't need a new line for each sentence, just a line break after 100 characters